040-SDK/102-create.mdx

```diff
@@ -354,9 +356,11 @@ users, _ := client.BulkInsert(context.TODO(), xata.BulkInsertRecordRequest{
 
 </TabbedCode>
 
-It is possible to provide IDs for each of the records. If records with the same IDs already exist, the bulk operation is cancelled and an error is returned.
+The bulk create method fails if the record ID already exists.
+
+It is possible to provide IDs for each of the records. If records with the same IDs already exist, the bulk operation is canceled and an error is returned. The bulk create method fails if the record ID already exists.
```

Comment:
"The bulk create method fails if the record ID already exists." is mentioned twice, in lines 359 and in 361. Should we clean up one of the two?

Reply:
Thanks! My error